http/dashboard_test.go

```diff
@@ -448,24 +448,26 @@ func TestService_handleGetDashboard(t *testing.T) {
       "x": 1,
       "y": 2,
       "w": 3,
-	  "h": 4,
-	  "name": "the cell name",
-	  "properties": {
-		"shape": "chronograf-v2",
-		"axes": null,
-		"colors": null,
-		"geom": "",
-		"legend": {},
-		"position": "",
-		"note": "",
-		"queries": null,
-		"shadeBelow": false,
-		"hoverDimension": "",
-		"showNoteWhenEmpty": false,
-		"timeFormat": "",
-		"type": "xy",
-		"xColumn": "",
-		"yColumn": ""
+			"h": 4,
+			"name": "the cell name",
+			"properties": {
+			"shape": "chronograf-v2",
+			"axes": null,
+			"colors": null,
+			"geom": "",
+			"legend": {},
+			"position": "",
+			"note": "",
+			"queries": null,
+			"shadeBelow": false,
+			"hoverDimension": "",
+			"showNoteWhenEmpty": false,
+			"timeFormat": "",
+			"type": "xy",
+			"xColumn": "",
+			"yColumn": "",
+			"legendOpacity": 0,
```

Comment:
does opacity default to 1 or 0?

Reply:
Opacity will default to 1. It is 0 here as a zero value. This is not a test for the "default" value. It is a test of the zero value.